Found 5 items
-rw-r--r--   3 root supergroup 1073741824 2019-03-20 15:22 /myfile1
-rw-r--r--   3 root supergroup 1073741824 2019-03-20 15:22 /myfile2
-rw-r--r--   3 root supergroup 1073741824 2019-03-20 15:22 /myfile3
-rw-r--r--   3 root supergroup 1073741824 2019-03-20 15:22 /myfile4
-rw-r--r--   3 root supergroup 1073741824 2019-03-20 15:22 /myfile5
file 4 has been put into HDFS
file 4 has been read from HDFS for 1 time
diff succeed!
file 4 has been read from HDFS for 2 time
diff succeed!
file 4 has been read from HDFS for 3 time
diff succeed!
file 4 has been read from HDFS for 4 time
diff succeed!
file 4 has been read from HDFS for 5 time
diff succeed!
file 4 has been read from HDFS for 6 time
diff succeed!
file 4 has been read from HDFS for 7 time
diff succeed!
file 4 has been read from HDFS for 8 time
diff succeed!
file 4 has been read from HDFS for 9 time
diff succeed!
file 4 has been read from HDFS for 10 time
diff succeed!
Deleted /myfile4
file 4 has been removed from HDFS
Found 5 items
-rw-r--r--   3 root supergroup 1073741824 2019-03-20 15:24 /myfile1
-rw-r--r--   3 root supergroup 1073741824 2019-03-20 15:24 /myfile2._COPYING_
-rw-r--r--   3 root supergroup 1073741824 2019-03-20 15:24 /myfile3._COPYING_
-rw-r--r--   3 root supergroup 1073741824 2019-03-20 15:24 /myfile4
-rw-r--r--   3 root supergroup 1073741824 2019-03-20 15:24 /myfile5
file 4 has been put into HDFS
file 4 has been read from HDFS for 1 time
diff succeed!
file 4 has been read from HDFS for 2 time
diff succeed!
file 4 has been read from HDFS for 3 time
diff succeed!
file 4 has been read from HDFS for 4 time
diff succeed!
file 4 has been read from HDFS for 5 time
diff succeed!
file 4 has been read from HDFS for 6 time
diff succeed!
file 4 has been read from HDFS for 7 time
diff succeed!
file 4 has been read from HDFS for 8 time
diff succeed!
file 4 has been read from HDFS for 9 time
diff succeed!
file 4 has been read from HDFS for 10 time
diff succeed!
Deleted /myfile4
file 4 has been removed from HDFS
Found 4 items
-rw-r--r--   3 root supergroup 1073741824 2019-03-20 15:26 /myfile1
-rw-r--r--   3 root supergroup          0 2019-03-20 15:26 /myfile2._COPYING_
-rw-r--r--   3 root supergroup 1073741824 2019-03-20 15:26 /myfile4
-rw-r--r--   3 root supergroup 1073741824 2019-03-20 15:25 /myfile5
file 4 has been put into HDFS
file 4 has been read from HDFS for 1 time
diff succeed!
file 4 has been read from HDFS for 2 time
diff succeed!
file 4 has been read from HDFS for 3 time
diff succeed!
file 4 has been read from HDFS for 4 time
diff succeed!
file 4 has been read from HDFS for 5 time
diff succeed!
file 4 has been read from HDFS for 6 time
diff succeed!
file 4 has been read from HDFS for 7 time
diff succeed!
file 4 has been read from HDFS for 8 time
diff succeed!
file 4 has been read from HDFS for 9 time
diff succeed!
file 4 has been read from HDFS for 10 time
diff succeed!
Deleted /myfile4
file 4 has been removed from HDFS
2019-03-20 15:27:46,790 INFO  retry.RetryInvocationHandler RetryInvocationHandler.java:log:411 - java.io.EOFException: End of File Exception between local host is: "clnode051.clemson.cloudlab.us/130.127.133.60"; destination host is: "node-1-link-0":8020; : java.io.EOFException; For more details see:  http://wiki.apache.org/hadoop/EOFException, while invoking ClientNamenodeProtocolTranslatorPB.addBlock over node-1-link-0/10.10.1.4:8020. Trying to failover immediately.
2019-03-20 15:27:46,798 INFO  retry.RetryInvocationHandler RetryInvocationHandler.java:log:411 - org.apache.hadoop.ipc.RemoteException(org.apache.hadoop.ipc.StandbyException): Operation category READ is not supported in state standby. Visit https://s.apache.org/sbnn-error
	at org.apache.hadoop.hdfs.server.namenode.ha.StandbyState.checkOperation(StandbyState.java:88)
	at org.apache.hadoop.hdfs.server.namenode.NameNode$NameNodeHAContext.checkOperation(NameNode.java:1952)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.checkOperation(FSNamesystem.java:1423)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.getAdditionalBlock(FSNamesystem.java:2674)
	at org.apache.hadoop.hdfs.server.namenode.NameNodeRpcServer.addBlock(NameNodeRpcServer.java:875)
	at org.apache.hadoop.hdfs.protocolPB.ClientNamenodeProtocolServerSideTranslatorPB.addBlock(ClientNamenodeProtocolServerSideTranslatorPB.java:561)
	at org.apache.hadoop.hdfs.protocol.proto.ClientNamenodeProtocolProtos$ClientNamenodeProtocol$2.callBlockingMethod(ClientNamenodeProtocolProtos.java)
	at org.apache.hadoop.ipc.ProtobufRpcEngine$Server$ProtoBufRpcInvoker.call(ProtobufRpcEngine.java:523)
	at org.apache.hadoop.ipc.RPC$Server.call(RPC.java:991)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:872)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:818)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:422)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1729)
	at org.apache.hadoop.ipc.Server$Handler.run(Server.java:2678)
, while invoking ClientNamenodeProtocolTranslatorPB.addBlock over node-0-link-0/10.10.1.1:8020 after 1 failover attempts. Trying to failover after sleeping for 808ms.
2019-03-20 15:27:47,613 INFO  retry.RetryInvocationHandler RetryInvocationHandler.java:log:411 - java.net.ConnectException: Call From clnode051.clemson.cloudlab.us/130.127.133.60 to node-1-link-0:8020 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking ClientNamenodeProtocolTranslatorPB.addBlock over node-1-link-0/10.10.1.4:8020 after 2 failover attempts. Trying to failover after sleeping for 1279ms.
2019-03-20 15:27:48,894 INFO  retry.RetryInvocationHandler RetryInvocationHandler.java:log:411 - org.apache.hadoop.ipc.RemoteException(org.apache.hadoop.ipc.StandbyException): Operation category READ is not supported in state standby. Visit https://s.apache.org/sbnn-error
	at org.apache.hadoop.hdfs.server.namenode.ha.StandbyState.checkOperation(StandbyState.java:88)
	at org.apache.hadoop.hdfs.server.namenode.NameNode$NameNodeHAContext.checkOperation(NameNode.java:1952)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.checkOperation(FSNamesystem.java:1423)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.getAdditionalBlock(FSNamesystem.java:2674)
	at org.apache.hadoop.hdfs.server.namenode.NameNodeRpcServer.addBlock(NameNodeRpcServer.java:875)
	at org.apache.hadoop.hdfs.protocolPB.ClientNamenodeProtocolServerSideTranslatorPB.addBlock(ClientNamenodeProtocolServerSideTranslatorPB.java:561)
	at org.apache.hadoop.hdfs.protocol.proto.ClientNamenodeProtocolProtos$ClientNamenodeProtocol$2.callBlockingMethod(ClientNamenodeProtocolProtos.java)
	at org.apache.hadoop.ipc.ProtobufRpcEngine$Server$ProtoBufRpcInvoker.call(ProtobufRpcEngine.java:523)
	at org.apache.hadoop.ipc.RPC$Server.call(RPC.java:991)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:872)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:818)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:422)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1729)
	at org.apache.hadoop.ipc.Server$Handler.run(Server.java:2678)
, while invoking ClientNamenodeProtocolTranslatorPB.addBlock over node-0-link-0/10.10.1.1:8020 after 3 failover attempts. Trying to failover after sleeping for 2610ms.
2019-03-20 15:27:51,507 INFO  retry.RetryInvocationHandler RetryInvocationHandler.java:log:411 - java.net.ConnectException: Call From clnode051.clemson.cloudlab.us/130.127.133.60 to node-1-link-0:8020 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking ClientNamenodeProtocolTranslatorPB.addBlock over node-1-link-0/10.10.1.4:8020 after 4 failover attempts. Trying to failover after sleeping for 4007ms.
Found 5 items
-rw-r--r--   3 root supergroup 1073741824 2019-03-20 15:27 /myfile1
-rw-r--r--   3 root supergroup 1073741824 2019-03-20 15:28 /myfile2
-rw-r--r--   3 root supergroup  671088640 2019-03-20 15:27 /myfile3._COPYING_
-rw-r--r--   3 root supergroup 1073741824 2019-03-20 15:27 /myfile4
-rw-r--r--   3 root supergroup 1073741824 2019-03-20 15:27 /myfile5
file 4 has been put into HDFS
file 4 has been read from HDFS for 1 time
diff succeed!
file 4 has been read from HDFS for 2 time
diff succeed!
2019-03-20 15:28:16,190 INFO  retry.RetryInvocationHandler RetryInvocationHandler.java:log:411 - org.apache.hadoop.ipc.RemoteException(org.apache.hadoop.ipc.StandbyException): Operation category READ is not supported in state standby. Visit https://s.apache.org/sbnn-error
	at org.apache.hadoop.hdfs.server.namenode.ha.StandbyState.checkOperation(StandbyState.java:88)
	at org.apache.hadoop.hdfs.server.namenode.NameNode$NameNodeHAContext.checkOperation(NameNode.java:1952)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.checkOperation(FSNamesystem.java:1423)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.getFileInfo(FSNamesystem.java:3085)
	at org.apache.hadoop.hdfs.server.namenode.NameNodeRpcServer.getFileInfo(NameNodeRpcServer.java:1154)
	at org.apache.hadoop.hdfs.protocolPB.ClientNamenodeProtocolServerSideTranslatorPB.getFileInfo(ClientNamenodeProtocolServerSideTranslatorPB.java:966)
	at org.apache.hadoop.hdfs.protocol.proto.ClientNamenodeProtocolProtos$ClientNamenodeProtocol$2.callBlockingMethod(ClientNamenodeProtocolProtos.java)
	at org.apache.hadoop.ipc.ProtobufRpcEngine$Server$ProtoBufRpcInvoker.call(ProtobufRpcEngine.java:523)
	at org.apache.hadoop.ipc.RPC$Server.call(RPC.java:991)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:872)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:818)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:422)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1729)
	at org.apache.hadoop.ipc.Server$Handler.run(Server.java:2678)
, while invoking ClientNamenodeProtocolTranslatorPB.getFileInfo over node-1-link-0/10.10.1.4:8020 after 1 failover attempts. Trying to failover after sleeping for 768ms.
2019-03-20 15:28:16,969 INFO  retry.RetryInvocationHandler RetryInvocationHandler.java:log:411 - java.net.ConnectException: Call From clnode051.clemson.cloudlab.us/130.127.133.60 to node-0-link-0:8020 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking ClientNamenodeProtocolTranslatorPB.getFileInfo over node-0-link-0/10.10.1.1:8020 after 2 failover attempts. Trying to failover after sleeping for 2775ms.
file 4 has been read from HDFS for 3 time
diff succeed!
file 4 has been read from HDFS for 4 time
diff succeed!
file 4 has been read from HDFS for 5 time
diff succeed!
file 4 has been read from HDFS for 6 time
diff succeed!
file 4 has been read from HDFS for 7 time
diff succeed!
file 4 has been read from HDFS for 8 time
diff succeed!
file 4 has been read from HDFS for 9 time
diff succeed!
file 4 has been read from HDFS for 10 time
diff succeed!
Deleted /myfile4
file 4 has been removed from HDFS
Found 5 items
-rw-r--r--   3 root supergroup 1073741824 2019-03-20 15:29 /myfile1
-rw-r--r--   3 root supergroup 1073741824 2019-03-20 15:29 /myfile2
-rw-r--r--   3 root supergroup 1073741824 2019-03-20 15:29 /myfile3
-rw-r--r--   3 root supergroup 1073741824 2019-03-20 15:29 /myfile4
-rw-r--r--   3 root supergroup 1073741824 2019-03-20 15:29 /myfile5
file 4 has been put into HDFS
file 4 has been read from HDFS for 1 time
diff succeed!
file 4 has been read from HDFS for 2 time
diff succeed!
file 4 has been read from HDFS for 3 time
diff succeed!
file 4 has been read from HDFS for 4 time
diff succeed!
file 4 has been read from HDFS for 5 time
diff succeed!
file 4 has been read from HDFS for 6 time
diff succeed!
file 4 has been read from HDFS for 7 time
diff succeed!
file 4 has been read from HDFS for 8 time
diff succeed!
file 4 has been read from HDFS for 9 time
diff succeed!
file 4 has been read from HDFS for 10 time
diff succeed!
Deleted /myfile4
file 4 has been removed from HDFS
Found 5 items
-rw-r--r--   3 root supergroup 1073741824 2019-03-20 15:31 /myfile1
-rw-r--r--   3 root supergroup 1073741824 2019-03-20 15:31 /myfile2
-rw-r--r--   3 root supergroup  939524096 2019-03-20 15:31 /myfile3._COPYING_
-rw-r--r--   3 root supergroup 1073741824 2019-03-20 15:31 /myfile4
-rw-r--r--   3 root supergroup 1073741824 2019-03-20 15:30 /myfile5
file 4 has been put into HDFS
file 4 has been read from HDFS for 1 time
diff succeed!
file 4 has been read from HDFS for 2 time
diff succeed!
file 4 has been read from HDFS for 3 time
diff succeed!
file 4 has been read from HDFS for 4 time
diff succeed!
file 4 has been read from HDFS for 5 time
diff succeed!
file 4 has been read from HDFS for 6 time
diff succeed!
file 4 has been read from HDFS for 7 time
diff succeed!
file 4 has been read from HDFS for 8 time
diff succeed!
file 4 has been read from HDFS for 9 time
diff succeed!
file 4 has been read from HDFS for 10 time
diff succeed!
Deleted /myfile4
file 4 has been removed from HDFS
Found 4 items
-rw-r--r--   3 root supergroup 1073741824 2019-03-20 15:32 /myfile1
-rw-r--r--   3 root supergroup 1073741824 2019-03-20 15:32 /myfile2
-rw-r--r--   3 root supergroup 1073741824 2019-03-20 15:32 /myfile4
-rw-r--r--   3 root supergroup 1073741824 2019-03-20 15:32 /myfile5
file 4 has been put into HDFS
file 4 has been read from HDFS for 1 time
diff succeed!
file 4 has been read from HDFS for 2 time
diff succeed!
file 4 has been read from HDFS for 3 time
diff succeed!
file 4 has been read from HDFS for 4 time
diff succeed!
file 4 has been read from HDFS for 5 time
diff succeed!
file 4 has been read from HDFS for 6 time
diff succeed!
file 4 has been read from HDFS for 7 time
diff succeed!
2019-03-20 15:33:57,826 INFO  retry.RetryInvocationHandler RetryInvocationHandler.java:log:411 - java.net.ConnectException: Call From clnode051.clemson.cloudlab.us/130.127.133.60 to node-1-link-0:8020 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking ClientNamenodeProtocolTranslatorPB.getFileInfo over node-1-link-0/10.10.1.4:8020 after 1 failover attempts. Trying to failover after sleeping for 765ms.
2019-03-20 15:33:58,598 INFO  retry.RetryInvocationHandler RetryInvocationHandler.java:log:411 - org.apache.hadoop.ipc.RemoteException(org.apache.hadoop.ipc.StandbyException): Operation category READ is not supported in state standby. Visit https://s.apache.org/sbnn-error
	at org.apache.hadoop.hdfs.server.namenode.ha.StandbyState.checkOperation(StandbyState.java:88)
	at org.apache.hadoop.hdfs.server.namenode.NameNode$NameNodeHAContext.checkOperation(NameNode.java:1952)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.checkOperation(FSNamesystem.java:1423)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.getFileInfo(FSNamesystem.java:3085)
	at org.apache.hadoop.hdfs.server.namenode.NameNodeRpcServer.getFileInfo(NameNodeRpcServer.java:1154)
	at org.apache.hadoop.hdfs.protocolPB.ClientNamenodeProtocolServerSideTranslatorPB.getFileInfo(ClientNamenodeProtocolServerSideTranslatorPB.java:966)
	at org.apache.hadoop.hdfs.protocol.proto.ClientNamenodeProtocolProtos$ClientNamenodeProtocol$2.callBlockingMethod(ClientNamenodeProtocolProtos.java)
	at org.apache.hadoop.ipc.ProtobufRpcEngine$Server$ProtoBufRpcInvoker.call(ProtobufRpcEngine.java:523)
	at org.apache.hadoop.ipc.RPC$Server.call(RPC.java:991)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:872)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:818)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:422)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1729)
	at org.apache.hadoop.ipc.Server$Handler.run(Server.java:2678)
, while invoking ClientNamenodeProtocolTranslatorPB.getFileInfo over node-0-link-0/10.10.1.1:8020 after 2 failover attempts. Trying to failover after sleeping for 2992ms.
2019-03-20 15:34:01,593 INFO  retry.RetryInvocationHandler RetryInvocationHandler.java:log:411 - java.net.ConnectException: Call From clnode051.clemson.cloudlab.us/130.127.133.60 to node-1-link-0:8020 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking ClientNamenodeProtocolTranslatorPB.getFileInfo over node-1-link-0/10.10.1.4:8020 after 3 failover attempts. Trying to failover after sleeping for 5545ms.
file 4 has been read from HDFS for 8 time
diff succeed!
file 4 has been read from HDFS for 9 time
diff succeed!
2019-03-20 15:34:22,754 INFO  retry.RetryInvocationHandler RetryInvocationHandler.java:log:411 - org.apache.hadoop.ipc.RemoteException(org.apache.hadoop.ipc.StandbyException): Operation category READ is not supported in state standby. Visit https://s.apache.org/sbnn-error
	at org.apache.hadoop.hdfs.server.namenode.ha.StandbyState.checkOperation(StandbyState.java:88)
	at org.apache.hadoop.hdfs.server.namenode.NameNode$NameNodeHAContext.checkOperation(NameNode.java:1952)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.checkOperation(FSNamesystem.java:1423)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.getFileInfo(FSNamesystem.java:3085)
	at org.apache.hadoop.hdfs.server.namenode.NameNodeRpcServer.getFileInfo(NameNodeRpcServer.java:1154)
	at org.apache.hadoop.hdfs.protocolPB.ClientNamenodeProtocolServerSideTranslatorPB.getFileInfo(ClientNamenodeProtocolServerSideTranslatorPB.java:966)
	at org.apache.hadoop.hdfs.protocol.proto.ClientNamenodeProtocolProtos$ClientNamenodeProtocol$2.callBlockingMethod(ClientNamenodeProtocolProtos.java)
	at org.apache.hadoop.ipc.ProtobufRpcEngine$Server$ProtoBufRpcInvoker.call(ProtobufRpcEngine.java:523)
	at org.apache.hadoop.ipc.RPC$Server.call(RPC.java:991)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:872)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:818)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:422)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1729)
	at org.apache.hadoop.ipc.Server$Handler.run(Server.java:2678)
, while invoking ClientNamenodeProtocolTranslatorPB.getFileInfo over node-1-link-0/10.10.1.4:8020 after 1 failover attempts. Trying to failover after sleeping for 817ms.
2019-03-20 15:34:23,580 INFO  retry.RetryInvocationHandler RetryInvocationHandler.java:log:411 - java.net.ConnectException: Call From clnode051.clemson.cloudlab.us/130.127.133.60 to node-0-link-0:8020 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking ClientNamenodeProtocolTranslatorPB.getFileInfo over node-0-link-0/10.10.1.1:8020 after 2 failover attempts. Trying to failover after sleeping for 1679ms.
file 4 has been read from HDFS for 10 time
diff succeed!
2019-03-20 15:34:34,016 INFO  retry.RetryInvocationHandler RetryInvocationHandler.java:log:411 - org.apache.hadoop.ipc.RemoteException(org.apache.hadoop.ipc.RetriableException): org.apache.hadoop.hdfs.server.namenode.SafeModeException: Cannot delete /myfile4. Name node is in safe mode.
The reported blocks 25 has reached the threshold 0.9990 of total blocks 25. The number of live datanodes 4 has reached the minimum number 0. In safe mode extension. Safe mode will be turned off automatically in 1 seconds. NamenodeHostName:node-1-link-0
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.checkNameNodeSafeMode(FSNamesystem.java:1441)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.delete(FSNamesystem.java:2986)
	at org.apache.hadoop.hdfs.server.namenode.NameNodeRpcServer.delete(NameNodeRpcServer.java:1095)
	at org.apache.hadoop.hdfs.protocolPB.ClientNamenodeProtocolServerSideTranslatorPB.delete(ClientNamenodeProtocolServerSideTranslatorPB.java:692)
	at org.apache.hadoop.hdfs.protocol.proto.ClientNamenodeProtocolProtos$ClientNamenodeProtocol$2.callBlockingMethod(ClientNamenodeProtocolProtos.java)
	at org.apache.hadoop.ipc.ProtobufRpcEngine$Server$ProtoBufRpcInvoker.call(ProtobufRpcEngine.java:523)
	at org.apache.hadoop.ipc.RPC$Server.call(RPC.java:991)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:872)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:818)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:422)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1729)
	at org.apache.hadoop.ipc.Server$Handler.run(Server.java:2678)
Caused by: org.apache.hadoop.hdfs.server.namenode.SafeModeException: Cannot delete /myfile4. Name node is in safe mode.
The reported blocks 25 has reached the threshold 0.9990 of total blocks 25. The number of live datanodes 4 has reached the minimum number 0. In safe mode extension. Safe mode will be turned off automatically in 1 seconds. NamenodeHostName:node-1-link-0
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.newSafemodeException(FSNamesystem.java:1450)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.checkNameNodeSafeMode(FSNamesystem.java:1437)
	... 12 more
, while invoking ClientNamenodeProtocolTranslatorPB.delete over node-1-link-0/10.10.1.4:8020. Retrying immediately.
2019-03-20 15:34:34,024 INFO  retry.RetryInvocationHandler RetryInvocationHandler.java:log:411 - org.apache.hadoop.ipc.RemoteException(org.apache.hadoop.ipc.RetriableException): org.apache.hadoop.hdfs.server.namenode.SafeModeException: Cannot delete /myfile4. Name node is in safe mode.
The reported blocks 25 has reached the threshold 0.9990 of total blocks 25. The number of live datanodes 4 has reached the minimum number 0. In safe mode extension. Safe mode will be turned off automatically in 1 seconds. NamenodeHostName:node-1-link-0
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.checkNameNodeSafeMode(FSNamesystem.java:1441)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.delete(FSNamesystem.java:2986)
	at org.apache.hadoop.hdfs.server.namenode.NameNodeRpcServer.delete(NameNodeRpcServer.java:1095)
	at org.apache.hadoop.hdfs.protocolPB.ClientNamenodeProtocolServerSideTranslatorPB.delete(ClientNamenodeProtocolServerSideTranslatorPB.java:692)
	at org.apache.hadoop.hdfs.protocol.proto.ClientNamenodeProtocolProtos$ClientNamenodeProtocol$2.callBlockingMethod(ClientNamenodeProtocolProtos.java)
	at org.apache.hadoop.ipc.ProtobufRpcEngine$Server$ProtoBufRpcInvoker.call(ProtobufRpcEngine.java:523)
	at org.apache.hadoop.ipc.RPC$Server.call(RPC.java:991)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:872)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:818)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:422)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1729)
	at org.apache.hadoop.ipc.Server$Handler.run(Server.java:2678)
Caused by: org.apache.hadoop.hdfs.server.namenode.SafeModeException: Cannot delete /myfile4. Name node is in safe mode.
The reported blocks 25 has reached the threshold 0.9990 of total blocks 25. The number of live datanodes 4 has reached the minimum number 0. In safe mode extension. Safe mode will be turned off automatically in 1 seconds. NamenodeHostName:node-1-link-0
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.newSafemodeException(FSNamesystem.java:1450)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.checkNameNodeSafeMode(FSNamesystem.java:1437)
	... 12 more
, while invoking ClientNamenodeProtocolTranslatorPB.delete over node-1-link-0/10.10.1.4:8020. Retrying after sleeping for 867ms.
2019-03-20 15:34:34,894 INFO  retry.RetryInvocationHandler RetryInvocationHandler.java:log:411 - org.apache.hadoop.ipc.RemoteException(org.apache.hadoop.ipc.RetriableException): org.apache.hadoop.hdfs.server.namenode.SafeModeException: Cannot delete /myfile4. Name node is in safe mode.
The reported blocks 25 has reached the threshold 0.9990 of total blocks 25. The number of live datanodes 4 has reached the minimum number 0. In safe mode extension. Safe mode will be turned off automatically in 0 seconds. NamenodeHostName:node-1-link-0
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.checkNameNodeSafeMode(FSNamesystem.java:1441)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.delete(FSNamesystem.java:2986)
	at org.apache.hadoop.hdfs.server.namenode.NameNodeRpcServer.delete(NameNodeRpcServer.java:1095)
	at org.apache.hadoop.hdfs.protocolPB.ClientNamenodeProtocolServerSideTranslatorPB.delete(ClientNamenodeProtocolServerSideTranslatorPB.java:692)
	at org.apache.hadoop.hdfs.protocol.proto.ClientNamenodeProtocolProtos$ClientNamenodeProtocol$2.callBlockingMethod(ClientNamenodeProtocolProtos.java)
	at org.apache.hadoop.ipc.ProtobufRpcEngine$Server$ProtoBufRpcInvoker.call(ProtobufRpcEngine.java:523)
	at org.apache.hadoop.ipc.RPC$Server.call(RPC.java:991)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:872)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:818)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:422)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1729)
	at org.apache.hadoop.ipc.Server$Handler.run(Server.java:2678)
Caused by: org.apache.hadoop.hdfs.server.namenode.SafeModeException: Cannot delete /myfile4. Name node is in safe mode.
The reported blocks 25 has reached the threshold 0.9990 of total blocks 25. The number of live datanodes 4 has reached the minimum number 0. In safe mode extension. Safe mode will be turned off automatically in 0 seconds. NamenodeHostName:node-1-link-0
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.newSafemodeException(FSNamesystem.java:1450)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.checkNameNodeSafeMode(FSNamesystem.java:1437)
	... 12 more
, while invoking ClientNamenodeProtocolTranslatorPB.delete over node-1-link-0/10.10.1.4:8020. Retrying after sleeping for 2255ms.
Deleted /myfile4
file 4 has been removed from HDFS
Found 5 items
-rw-r--r--   3 root supergroup  134217728 2019-03-20 15:34 /myfile1._COPYING_
-rw-r--r--   3 root supergroup 1073741824 2019-03-20 15:34 /myfile2
-rw-r--r--   3 root supergroup 1073741824 2019-03-20 15:34 /myfile3
-rw-r--r--   3 root supergroup 1073741824 2019-03-20 15:34 /myfile4
-rw-r--r--   3 root supergroup 1073741824 2019-03-20 15:34 /myfile5
file 4 has been put into HDFS
file 4 has been read from HDFS for 1 time
diff succeed!
file 4 has been read from HDFS for 2 time
diff succeed!
file 4 has been read from HDFS for 3 time
diff succeed!
file 4 has been read from HDFS for 4 time
diff succeed!
file 4 has been read from HDFS for 5 time
diff succeed!
file 4 has been read from HDFS for 6 time
diff succeed!
file 4 has been read from HDFS for 7 time
diff succeed!
file 4 has been read from HDFS for 8 time
diff succeed!
file 4 has been read from HDFS for 9 time
diff succeed!
file 4 has been read from HDFS for 10 time
diff succeed!
Deleted /myfile4
file 4 has been removed from HDFS
Found 5 items
-rw-r--r--   3 root supergroup  671088640 2019-03-20 15:36 /myfile1._COPYING_
-rw-r--r--   3 root supergroup  671088640 2019-03-20 15:36 /myfile2._COPYING_
-rw-r--r--   3 root supergroup 1073741824 2019-03-20 15:36 /myfile3
-rw-r--r--   3 root supergroup 1073741824 2019-03-20 15:36 /myfile4
-rw-r--r--   3 root supergroup 1073741824 2019-03-20 15:35 /myfile5
file 4 has been put into HDFS
file 4 has been read from HDFS for 1 time
diff succeed!
file 4 has been read from HDFS for 2 time
diff succeed!
file 4 has been read from HDFS for 3 time
diff succeed!
file 4 has been read from HDFS for 4 time
diff succeed!
file 4 has been read from HDFS for 5 time
diff succeed!
file 4 has been read from HDFS for 6 time
diff succeed!
file 4 has been read from HDFS for 7 time
diff succeed!
file 4 has been read from HDFS for 8 time
diff succeed!
file 4 has been read from HDFS for 9 time
diff succeed!
file 4 has been read from HDFS for 10 time
diff succeed!
Deleted /myfile4
file 4 has been removed from HDFS
Found 5 items
-rw-r--r--   3 root supergroup          0 2019-03-20 15:37 /myfile1._COPYING_
-rw-r--r--   3 root supergroup  134217728 2019-03-20 15:37 /myfile2._COPYING_
-rw-r--r--   3 root supergroup 1073741824 2019-03-20 15:37 /myfile3
-rw-r--r--   3 root supergroup 1073741824 2019-03-20 15:37 /myfile4
-rw-r--r--   3 root supergroup 1073741824 2019-03-20 15:37 /myfile5
file 4 has been put into HDFS
file 4 has been read from HDFS for 1 time
diff succeed!
file 4 has been read from HDFS for 2 time
diff succeed!
file 4 has been read from HDFS for 3 time
diff succeed!
file 4 has been read from HDFS for 4 time
diff succeed!
file 4 has been read from HDFS for 5 time
diff succeed!
file 4 has been read from HDFS for 6 time
diff succeed!
file 4 has been read from HDFS for 7 time
diff succeed!
file 4 has been read from HDFS for 8 time
diff succeed!
file 4 has been read from HDFS for 9 time
diff succeed!
file 4 has been read from HDFS for 10 time
diff succeed!
Deleted /myfile4
file 4 has been removed from HDFS
Found 5 items
-rw-r--r--   3 root supergroup 1073741824 2019-03-20 15:38 /myfile1
-rw-r--r--   3 root supergroup 1073741824 2019-03-20 15:38 /myfile2
-rw-r--r--   3 root supergroup 1073741824 2019-03-20 15:39 /myfile3
-rw-r--r--   3 root supergroup 1073741824 2019-03-20 15:39 /myfile4
-rw-r--r--   3 root supergroup 1073741824 2019-03-20 15:39 /myfile5
file 4 has been put into HDFS
file 4 has been read from HDFS for 1 time
diff succeed!
file 4 has been read from HDFS for 2 time
diff succeed!
file 4 has been read from HDFS for 3 time
diff succeed!
file 4 has been read from HDFS for 4 time
diff succeed!
file 4 has been read from HDFS for 5 time
diff succeed!
file 4 has been read from HDFS for 6 time
diff succeed!
file 4 has been read from HDFS for 7 time
diff succeed!
file 4 has been read from HDFS for 8 time
diff succeed!
file 4 has been read from HDFS for 9 time
diff succeed!
file 4 has been read from HDFS for 10 time
diff succeed!
Deleted /myfile4
file 4 has been removed from HDFS
Found 5 items
-rw-r--r--   3 root supergroup 1073741824 2019-03-20 15:39 /myfile1
-rw-r--r--   3 root supergroup 1073741824 2019-03-20 15:39 /myfile2
-rw-r--r--   3 root supergroup 1073741824 2019-03-20 15:40 /myfile3
-rw-r--r--   3 root supergroup 1073741824 2019-03-20 15:40 /myfile4
-rw-r--r--   3 root supergroup 1073741824 2019-03-20 15:40 /myfile5
file 4 has been put into HDFS
file 4 has been read from HDFS for 1 time
diff succeed!
file 4 has been read from HDFS for 2 time
diff succeed!
file 4 has been read from HDFS for 3 time
diff succeed!
file 4 has been read from HDFS for 4 time
diff succeed!
file 4 has been read from HDFS for 5 time
diff succeed!
file 4 has been read from HDFS for 6 time
diff succeed!
file 4 has been read from HDFS for 7 time
diff succeed!
file 4 has been read from HDFS for 8 time
diff succeed!
file 4 has been read from HDFS for 9 time
diff succeed!
file 4 has been read from HDFS for 10 time
diff succeed!
Deleted /myfile4
file 4 has been removed from HDFS
Found 5 items
-rw-r--r--   3 root supergroup 1073741824 2019-03-20 15:41 /myfile1
-rw-r--r--   3 root supergroup 1073741824 2019-03-20 15:41 /myfile2
-rw-r--r--   3 root supergroup 1073741824 2019-03-20 15:42 /myfile3
-rw-r--r--   3 root supergroup 1073741824 2019-03-20 15:42 /myfile4
-rw-r--r--   3 root supergroup 1073741824 2019-03-20 15:42 /myfile5
file 4 has been put into HDFS
file 4 has been read from HDFS for 1 time
diff succeed!
file 4 has been read from HDFS for 2 time
diff succeed!
file 4 has been read from HDFS for 3 time
diff succeed!
file 4 has been read from HDFS for 4 time
diff succeed!
file 4 has been read from HDFS for 5 time
diff succeed!
file 4 has been read from HDFS for 6 time
diff succeed!
file 4 has been read from HDFS for 7 time
diff succeed!
file 4 has been read from HDFS for 8 time
diff succeed!
file 4 has been read from HDFS for 9 time
diff succeed!
file 4 has been read from HDFS for 10 time
diff succeed!
Deleted /myfile4
file 4 has been removed from HDFS
Found 5 items
-rw-r--r--   3 root supergroup 1073741824 2019-03-20 15:43 /myfile1
-rw-r--r--   3 root supergroup 1073741824 2019-03-20 15:43 /myfile2
-rw-r--r--   3 root supergroup 1073741824 2019-03-20 15:44 /myfile3
-rw-r--r--   3 root supergroup 1073741824 2019-03-20 15:44 /myfile4
-rw-r--r--   3 root supergroup 1073741824 2019-03-20 15:44 /myfile5
file 4 has been put into HDFS
file 4 has been read from HDFS for 1 time
diff succeed!
file 4 has been read from HDFS for 2 time
diff succeed!
file 4 has been read from HDFS for 3 time
diff succeed!
file 4 has been read from HDFS for 4 time
diff succeed!
file 4 has been read from HDFS for 5 time
diff succeed!
file 4 has been read from HDFS for 6 time
diff succeed!
file 4 has been read from HDFS for 7 time
diff succeed!
file 4 has been read from HDFS for 8 time
diff succeed!
file 4 has been read from HDFS for 9 time
diff succeed!
2019-03-20 15:45:23,080 INFO  retry.RetryInvocationHandler RetryInvocationHandler.java:log:411 - java.net.ConnectException: Call From clnode051.clemson.cloudlab.us/130.127.133.60 to node-1-link-0:8020 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking ClientNamenodeProtocolTranslatorPB.getFileInfo over node-1-link-0/10.10.1.4:8020 after 1 failover attempts. Trying to failover after sleeping for 1252ms.
file 4 has been read from HDFS for 10 time
diff succeed!
Deleted /myfile4
file 4 has been removed from HDFS
2019-03-20 15:45:47,888 INFO  retry.RetryInvocationHandler RetryInvocationHandler.java:log:411 - java.io.EOFException: End of File Exception between local host is: "clnode051.clemson.cloudlab.us/130.127.133.60"; destination host is: "node-0-link-0":8020; : java.io.EOFException; For more details see:  http://wiki.apache.org/hadoop/EOFException, while invoking ClientNamenodeProtocolTranslatorPB.addBlock over node-0-link-0/10.10.1.1:8020. Trying to failover immediately.
2019-03-20 15:45:47,920 INFO  retry.RetryInvocationHandler RetryInvocationHandler.java:log:411 - org.apache.hadoop.ipc.RemoteException(org.apache.hadoop.ipc.StandbyException): Operation category READ is not supported in state standby. Visit https://s.apache.org/sbnn-error
	at org.apache.hadoop.hdfs.server.namenode.ha.StandbyState.checkOperation(StandbyState.java:88)
	at org.apache.hadoop.hdfs.server.namenode.NameNode$NameNodeHAContext.checkOperation(NameNode.java:1952)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.checkOperation(FSNamesystem.java:1423)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.getAdditionalBlock(FSNamesystem.java:2674)
	at org.apache.hadoop.hdfs.server.namenode.NameNodeRpcServer.addBlock(NameNodeRpcServer.java:875)
	at org.apache.hadoop.hdfs.protocolPB.ClientNamenodeProtocolServerSideTranslatorPB.addBlock(ClientNamenodeProtocolServerSideTranslatorPB.java:561)
	at org.apache.hadoop.hdfs.protocol.proto.ClientNamenodeProtocolProtos$ClientNamenodeProtocol$2.callBlockingMethod(ClientNamenodeProtocolProtos.java)
	at org.apache.hadoop.ipc.ProtobufRpcEngine$Server$ProtoBufRpcInvoker.call(ProtobufRpcEngine.java:523)
	at org.apache.hadoop.ipc.RPC$Server.call(RPC.java:991)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:872)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:818)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:422)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1729)
	at org.apache.hadoop.ipc.Server$Handler.run(Server.java:2678)
, while invoking ClientNamenodeProtocolTranslatorPB.addBlock over node-1-link-0/10.10.1.4:8020 after 1 failover attempts. Trying to failover after sleeping for 685ms.
2019-03-20 15:45:48,611 INFO  retry.RetryInvocationHandler RetryInvocationHandler.java:log:411 - java.net.ConnectException: Call From clnode051.clemson.cloudlab.us/130.127.133.60 to node-0-link-0:8020 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking ClientNamenodeProtocolTranslatorPB.addBlock over node-0-link-0/10.10.1.1:8020 after 2 failover attempts. Trying to failover after sleeping for 1771ms.
2019-03-20 15:45:50,386 INFO  retry.RetryInvocationHandler RetryInvocationHandler.java:log:411 - org.apache.hadoop.ipc.RemoteException(org.apache.hadoop.ipc.RetriableException): org.apache.hadoop.hdfs.server.namenode.SafeModeException: Cannot add block to /myfile4._COPYING_. Name node is in safe mode.
The reported blocks 16 has reached the threshold 0.9990 of total blocks 16. The number of live datanodes 4 has reached the minimum number 0. In safe mode extension. Safe mode will be turned off automatically in 10 seconds. NamenodeHostName:node-1-link-0
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.checkNameNodeSafeMode(FSNamesystem.java:1441)
	at org.apache.hadoop.hdfs.server.namenode.FSDirWriteFileOp.analyzeFileState(FSDirWriteFileOp.java:591)
	at org.apache.hadoop.hdfs.server.namenode.FSDirWriteFileOp.validateAddBlock(FSDirWriteFileOp.java:172)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.getAdditionalBlock(FSNamesystem.java:2679)
	at org.apache.hadoop.hdfs.server.namenode.NameNodeRpcServer.addBlock(NameNodeRpcServer.java:875)
	at org.apache.hadoop.hdfs.protocolPB.ClientNamenodeProtocolServerSideTranslatorPB.addBlock(ClientNamenodeProtocolServerSideTranslatorPB.java:561)
	at org.apache.hadoop.hdfs.protocol.proto.ClientNamenodeProtocolProtos$ClientNamenodeProtocol$2.callBlockingMethod(ClientNamenodeProtocolProtos.java)
	at org.apache.hadoop.ipc.ProtobufRpcEngine$Server$ProtoBufRpcInvoker.call(ProtobufRpcEngine.java:523)
	at org.apache.hadoop.ipc.RPC$Server.call(RPC.java:991)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:872)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:818)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:422)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1729)
	at org.apache.hadoop.ipc.Server$Handler.run(Server.java:2678)
Caused by: org.apache.hadoop.hdfs.server.namenode.SafeModeException: Cannot add block to /myfile4._COPYING_. Name node is in safe mode.
The reported blocks 16 has reached the threshold 0.9990 of total blocks 16. The number of live datanodes 4 has reached the minimum number 0. In safe mode extension. Safe mode will be turned off automatically in 10 seconds. NamenodeHostName:node-1-link-0
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.newSafemodeException(FSNamesystem.java:1450)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.checkNameNodeSafeMode(FSNamesystem.java:1437)
	... 14 more
, while invoking ClientNamenodeProtocolTranslatorPB.addBlock over node-1-link-0/10.10.1.4:8020 after 3 failover attempts. Retrying after sleeping for 3760ms.
2019-03-20 15:45:54,149 INFO  retry.RetryInvocationHandler RetryInvocationHandler.java:log:411 - org.apache.hadoop.ipc.RemoteException(org.apache.hadoop.ipc.RetriableException): org.apache.hadoop.hdfs.server.namenode.SafeModeException: Cannot add block to /myfile4._COPYING_. Name node is in safe mode.
The reported blocks 16 has reached the threshold 0.9990 of total blocks 16. The number of live datanodes 4 has reached the minimum number 0. In safe mode extension. Safe mode will be turned off automatically in 6 seconds. NamenodeHostName:node-1-link-0
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.checkNameNodeSafeMode(FSNamesystem.java:1441)
	at org.apache.hadoop.hdfs.server.namenode.FSDirWriteFileOp.analyzeFileState(FSDirWriteFileOp.java:591)
	at org.apache.hadoop.hdfs.server.namenode.FSDirWriteFileOp.validateAddBlock(FSDirWriteFileOp.java:172)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.getAdditionalBlock(FSNamesystem.java:2679)
	at org.apache.hadoop.hdfs.server.namenode.NameNodeRpcServer.addBlock(NameNodeRpcServer.java:875)
	at org.apache.hadoop.hdfs.protocolPB.ClientNamenodeProtocolServerSideTranslatorPB.addBlock(ClientNamenodeProtocolServerSideTranslatorPB.java:561)
	at org.apache.hadoop.hdfs.protocol.proto.ClientNamenodeProtocolProtos$ClientNamenodeProtocol$2.callBlockingMethod(ClientNamenodeProtocolProtos.java)
	at org.apache.hadoop.ipc.ProtobufRpcEngine$Server$ProtoBufRpcInvoker.call(ProtobufRpcEngine.java:523)
	at org.apache.hadoop.ipc.RPC$Server.call(RPC.java:991)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:872)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:818)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:422)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1729)
	at org.apache.hadoop.ipc.Server$Handler.run(Server.java:2678)
Caused by: org.apache.hadoop.hdfs.server.namenode.SafeModeException: Cannot add block to /myfile4._COPYING_. Name node is in safe mode.
The reported blocks 16 has reached the threshold 0.9990 of total blocks 16. The number of live datanodes 4 has reached the minimum number 0. In safe mode extension. Safe mode will be turned off automatically in 6 seconds. NamenodeHostName:node-1-link-0
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.newSafemodeException(FSNamesystem.java:1450)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.checkNameNodeSafeMode(FSNamesystem.java:1437)
	... 14 more
, while invoking ClientNamenodeProtocolTranslatorPB.addBlock over node-1-link-0/10.10.1.4:8020 after 3 failover attempts. Retrying after sleeping for 8922ms.
Found 5 items
-rw-r--r--   3 root supergroup 1073741824 2019-03-20 15:45 /myfile1
-rw-r--r--   3 root supergroup 1073741824 2019-03-20 15:45 /myfile2
-rw-r--r--   3 root supergroup  671088640 2019-03-20 15:45 /myfile3._COPYING_
-rw-r--r--   3 root supergroup 1073741824 2019-03-20 15:46 /myfile4
-rw-r--r--   3 root supergroup          0 2019-03-20 15:45 /myfile5._COPYING_
file 4 has been put into HDFS
file 4 has been read from HDFS for 1 time
diff succeed!
file 4 has been read from HDFS for 2 time
diff succeed!
file 4 has been read from HDFS for 3 time
diff succeed!
file 4 has been read from HDFS for 4 time
diff succeed!
file 4 has been read from HDFS for 5 time
diff succeed!
file 4 has been read from HDFS for 6 time
diff succeed!
file 4 has been read from HDFS for 7 time
diff succeed!
file 4 has been read from HDFS for 8 time
diff succeed!
file 4 has been read from HDFS for 9 time
diff succeed!
file 4 has been read from HDFS for 10 time
diff succeed!
Deleted /myfile4
file 4 has been removed from HDFS
Found 5 items
-rw-r--r--   3 root supergroup 1073741824 2019-03-20 15:46 /myfile1
-rw-r--r--   3 root supergroup 1073741824 2019-03-20 15:46 /myfile2
-rw-r--r--   3 root supergroup  402653184 2019-03-20 15:47 /myfile3._COPYING_
-rw-r--r--   3 root supergroup 1073741824 2019-03-20 15:47 /myfile4
-rw-r--r--   3 root supergroup 1073741824 2019-03-20 15:46 /myfile5
file 4 has been put into HDFS
file 4 has been read from HDFS for 1 time
diff succeed!
file 4 has been read from HDFS for 2 time
diff succeed!
file 4 has been read from HDFS for 3 time
diff succeed!
file 4 has been read from HDFS for 4 time
diff succeed!
file 4 has been read from HDFS for 5 time
diff succeed!
file 4 has been read from HDFS for 6 time
diff succeed!
file 4 has been read from HDFS for 7 time
diff succeed!
file 4 has been read from HDFS for 8 time
diff succeed!
file 4 has been read from HDFS for 9 time
diff succeed!
file 4 has been read from HDFS for 10 time
diff succeed!
Deleted /myfile4
file 4 has been removed from HDFS
Found 5 items
-rw-r--r--   3 root supergroup 1073741824 2019-03-20 15:48 /myfile1
-rw-r--r--   3 root supergroup 1073741824 2019-03-20 15:48 /myfile2
-rw-r--r--   3 root supergroup 1073741824 2019-03-20 15:49 /myfile3
-rw-r--r--   3 root supergroup 1073741824 2019-03-20 15:49 /myfile4
-rw-r--r--   3 root supergroup 1073741824 2019-03-20 15:48 /myfile5
file 4 has been put into HDFS
file 4 has been read from HDFS for 1 time
diff succeed!
file 4 has been read from HDFS for 2 time
diff succeed!
file 4 has been read from HDFS for 3 time
diff succeed!
file 4 has been read from HDFS for 4 time
diff succeed!
file 4 has been read from HDFS for 5 time
diff succeed!
file 4 has been read from HDFS for 6 time
diff succeed!
file 4 has been read from HDFS for 7 time
diff succeed!
file 4 has been read from HDFS for 8 time
diff succeed!
file 4 has been read from HDFS for 9 time
diff succeed!
file 4 has been read from HDFS for 10 time
diff succeed!
Deleted /myfile4
file 4 has been removed from HDFS
